utils/yoke/derive/examples/derive.rs

```diff
@@ -51,12 +51,12 @@ pub struct HasTuples<'data> {
 }
 
 pub fn assert_zcf_tuples<'b, 'data>(x: &'b HasTuples<'data>) -> HasTuples<'b> {
-    HasTuples::<'static>::zero_copy_from(x)
+    HasTuples::<'b>::zero_copy_from(x)
```

Comment:
question: is this explicit lt still necessary? one of the benefits of the simplification is that a bunch of these won't be needed anymore